<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-5KMFOFH9" failures="0" tests="3" name="com.SwagLabs.TestCases.CheckOutPageTest" time="2.794" errors="1" timestamp="2024-04-05T10:46:15 IST" skipped="2">
  <testcase classname="com.SwagLabs.TestCases.CheckOutPageTest" name="setUpBrowser" time="2.794">
    <error message="Cannot invoke &quot;com.aventstack.extentreports.ExtentTest.pass(String)&quot; because &quot;this.test&quot; is null" type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "com.aventstack.extentreports.ExtentTest.pass(String)" because "this.test" is null
at com.SwagLabs.TestCases.BaseClass.setUpBrowser(BaseClass.java:64)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:69)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:361)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:296)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:644)
at org.testng.TestRunner.beforeRun(TestRunner.java:633)
at org.testng.TestRunner.run(TestRunner.java:595)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- setUpBrowser -->
  <system-out>
    <![CDATA[Log: Driver session is created
Log: Apllication is open]]>
  </system-out>
  <testcase classname="com.SwagLabs.TestCases.CheckOutPageTest" name="verifyCancel" time="0.000">
    <skipped/>
  </testcase> <!-- verifyCancel -->
  <system-out/>
  <testcase classname="com.SwagLabs.TestCases.CheckOutPageTest" name="verifyCheckout" time="0.000">
    <skipped/>
  </testcase> <!-- verifyCheckout -->
  <system-out/>
</testsuite> <!-- com.SwagLabs.TestCases.CheckOutPageTest -->
